asciimatics/screen.py

```diff
@@ -1914,6 +1914,7 @@ def close(self, restore=True):
                 curses.echo()
                 curses.nocbreak()
                 curses.endwin()
+                signal.signal(signal.SIGWINCH, signal.SIG_DFL)
```

Comment:
So it looks like on ScreenResizeError. SIGWICH will be called which reinitializes curses, and then close will be called (which closes curses again). And then open will be called which initializes curses. Is that okay?